docs/api-ref.md

```diff
@@ -1374,6 +1374,55 @@ Returns the `JobItem` requested.
 ```
 
 
+<br>
+<br>
+
+---
+
+## Metadata
+
+Using the `metadata` endpoint, one can use GraphQL to query metadata on, e.g., workbooks, datasources, the fields contained within them, and the relationships between them.
+
+### Metadata methods
+
+
+The Metadata methods are based upon the [Metadata API](https://help.tableau.com/current/api/metadata_api/en-us/index.html) REST methods.
+
+
+Source files: server/endpoint/metadata_endpoint.py
+
+<br>
+<br>
+
+
+#### metadata.query
+
+
+```py
+metadata.query(query, variables=None, abort_on_error=False)
+```
+
+Executes a metadata query and returns the result.
+
+
+**Parameters**
+
+Name | Description
+:--- | :---
+`query`          | The GraphQL query
+`variables`      | Variables for the GraphQL query
+`abort_on_error` | Whether to throw an exception on error
+
+
+**Returns**
+
+The query results as a structure of `dict`s and `list`s
```

Comment:
The query results are json interpreted by the `json` package into dicts and lists, to be a little more explicit?